### PR TITLE
allow to pass configuration to babel

### LINKED ISSUE #7: Add plugin prefix to transpilled modules as well
I trying to use this plugin to process my es6 code on-the-fly. For example:

app.js
```js
import B from './B.js'
//other app code
```
B.js
```js
import C from './C.js'
//some code
```

In that case only `app.js` will be transplilled. Then require tries to load `B.js` and will get an error because it will be loaded without transpilling.

### PR DESCRIPTION
**Pull request by just-boris**
Babel have [a lot of options](http://babeljs.io/docs/usage/options/) and it would be good to have an ability to set them.

I've updated the demo where you can see how to use it. Also it help me to fix #7 because now I can provide desired option into babel and any other still will have the default behavior

### CODE REVIEW

**Comment by just-boris** [comment]
@mikach ping

File: demo/index.html [hunk 1]
```diff
@@ -10,6 +10,13 @@
     <script src="libs/requirejs.min.js"></script>
     <script>
         requirejs.config({
+            config: {
```
Comment (mikach) [inline demo/index.html]: I think this also must be included in `demo/build.config.js`. Check if optimisation works with `npm run optimize` and open `demo/index-optimized.html` in browser.

File: es6.js [hunk 2]
```diff
@@ -27,24 +27,35 @@ if (typeof window !== "undefined" && window.navigator && window.document) {
...
-    'babel'
+    'babel',
     //>>excludeEnd('excludeBabel')
+    'module'
 ], function(
     //>>excludeStart('excludeBabel', pragmas.excludeBabel)
-    babel
+    babel,
     //>>excludeEnd('excludeBabel')
+    module
```
Comment (mikach) [inline es6.js]: Maybe we need to rename this with `_module` (or something else) to exclude futures conflicts with native modules.

**Comment by mikach** [comment]
@just-boris sorry for delay. 

Thanks for you work! Check my comment for your changes.

**Comment by just-boris** [comment]
Fixed

**Comment by mikach** [comment]
Thanks!

**Pull request merged**